schemas/context/pushdetail.schema.json

```diff
@@ -0,0 +1,54 @@
+{
+    "meta:license": [
+        "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+        "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+        "you may not use this file except in compliance with the License. You may obtain a copy",
+        "of the License at https://creativecommons.org/licenses/by/4.0/"
+    ],
+    "$id": "https://ns.adobe.com/xdm/context/pushdetail",
+    "$schema": "http://json-schema.org/draft-06/schema#",
+    "title": "Push notification detail",
+    "type": "object",
+    "meta:extensible": true,
+    "description": "Push notification detail required to send mobile push notification to a specific mobile application on a given device.",
+    "definitions": {
+        "pushdetail": {
+            "properties": {
+                "xdm:token": {
+                    "title": "Push Token",
+                    "type": "string",
+                    "description": "Valid push token for the given push platform."
+                },
+                "xdm:platform": {
+                    "title": "Push Platform",
+                    "type": "string",
+                    "description": "Push provider, e.g. apns or fcm"
```

Comment:
Are there only two possible values?

Reply:
for now the possible values are fcm, apns, and apns-sandbox. This list might grow in the future.